deebot_client/commands/json/wash_info.py

```diff
@@ -0,0 +1,66 @@
+"""WashInfo command module."""
+
+from types import MappingProxyType
+from typing import Any
+
+from deebot_client.command import InitParam
+from deebot_client.event_bus import EventBus
+from deebot_client.events import WashInfoEvent, WashMode
+from deebot_client.message import HandlingResult
+
+from .common import JsonGetCommand, JsonSetCommand
+
+
+class GetWashInfo(JsonGetCommand):
```

Comment:
```suggestion
class GetWashInfo(OnWaterInfo, JsonGetCommand):
```

We don't need to duplicate the code